src/boot/cic6105.c

```diff
@@ -1,3 +1,85 @@
+#include "global.h"
 #include "cic6105.h"
+#include "fault.h"
 
-u32 B_80008EE0;
+// TODO N64 fault.c functions
+void func_800AE1E0_unknown(s32, s32);
+void func_800AE258_unknown(const char*, ...);
+
+s32 func_80001714(void);
+
+extern u64 cic6105ucodeTextStart[];
+
+OSTask D_800067C0_unknown = {
+    4, 0, rspbootTextStart, 0x3E8, cic6105ucodeTextStart, 0x20, (u64*)gBuildTeam, 8, NULL, 0, NULL, 0, NULL, 0, NULL, 0,
+};
+
+s32 B_80008EE0;
+s32 B_80008EE4;
+FaultClient sCIC6105FaultClient;
+s32 B_80008EF8;
+s32 B_80008EFC;
+
+void func_800014D0(void) {
+    R_AUDIOMGR_DEBUG_LEVEL = AUDIOMGR_DEBUG_LEVEL_NO_RSP;
+}
+
+void func_800014E8(void) {
+    R_AUDIOMGR_DEBUG_LEVEL = AUDIOMGR_DEBUG_LEVEL_NONE;
+}
+
+void CIC6105_FaultClient(void) {
+    s32 spStatus;
+
+    spStatus = IO_READ(SP_STATUS_REG);
+    func_800AE1E0_unknown(48, 200);
+    if (spStatus & SP_STATUS_SIG7) {
+        func_800AE258_unknown("OCARINA %08x %08x", B_80008EF8, B_80008EFC);
+    } else {
+        func_800AE258_unknown("LEGEND %08x %08x", B_80008EF8, B_80008EFC);
+    }
+    func_800AE1E0_unknown(40, 184);
+    func_800AE258_unknown("ROM_F");
+    func_800AE258_unknown(" [Creator:%s]", gBuildTeam);
```

Comment:
this is a super minor point, and not even related to your changes, but we should probably rename `gBuildTeam` to `gBuildCreator` at some point. The point is to identify exactly who built the rom, for example `zelda@srd44` and `zelda@srd022j` likely reference individual computer names/users at srd (the team)

feel free to ignore for now but want to say it while im thinking about it :)

Reply:
this is supported by the ique strings as well, which are non-srd
`build@toad.routefree.com 03-10-22 16:23:19 ` username "build" @ hostname "toad.routfree.com"
`tyu@linuxdev3 06-10-13 14:17:43` username "tyu" @ hostname "linuxdev3"